paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -328,13 +331,13 @@ internal class PaymentSheetViewModel @Inject internal constructor(
         }
 
         result.fold(
-            onSuccess = ::handlePaymentSheetStateLoaded,
-            onFailure = ::handlePaymentSheetStateLoadFailure,
+            onSuccess = { handlePaymentSheetStateLoaded(it) },
+            onFailure = { handlePaymentSheetStateLoadFailure(it) },
```

Comment:
Can these functions no longer be passed in as references due to them being suspendable?

Reply:
Yep, that’s the reason. Do you know of a remedy or just asking?

Reply:
Just asking. Given adding the `suspend` modifier changes the function signature there probably isn't a remedy for this.